### PR TITLE
es7210: Add I2C address to header.

### PR DESCRIPTION
**Pull request by espzav**
Added I2C addresses into header file.

### CODE REVIEW

**Review by tore-espressif — APPROVED** [review]
LGTM!

File: components/es7210/include/es7210.h [hunk 1]
```diff
@@ -9,6 +9,28 @@
...
+ *
+ * The 8-bit address format is as follows:
+ *
+ *                (Slave Address)
+ *     ┌─────────────────┷─────────────────┐
+ *  ┌─────┐─────┐─────┐─────┐─────┐─────┐─────┐─────┐
+ *  |  1  |  0  |  0  |  0  |  0  | AD1 | AD0 | R/W |
+ *  └─────┘─────┘─────┘─────┘─────┘─────┘─────┘─────┘
+ *     └────────┯────────┘           └───┯───┘
+ *           (Fixed)          (Hareware Selectable)
```
Comment (igrr) [inline components/es7210/include/es7210.h:23]:
```suggestion
 *           (Fixed)          (Hardware Selectable)
```

File: components/es7210/include/es7210.h [hunk 2]
```diff
@@ -9,6 +9,28 @@
...
+ *
+ * And the 7-bit slave address is the most important data for users.
+ * For example, if a chip's AD0,AD1 are connected to GND, it's 7-bit slave address is 1000000b(0x40).
```
Comment (igrr) [inline components/es7210/include/es7210.h:26]:
```suggestion
 * For example, if the chip's AD0,AD1 are connected to GND, its 7-bit slave address is 1000000b(0x40).
```

**Comment by espzav** [comment]
@igrr and @tore-espressif Thank you for the review!

**Pull request merged**